apps/src/music/blockly/themes.js

```diff
@@ -6,7 +6,7 @@ const blockStyles = cdoBlockStyles;
 const categoryStyles = {};
 const componentStyles = {
   toolboxBackgroundColour: '#424242', // gray-800
-  flyoutBackgroundColour: '#9e9e9e', // gray-500
+  flyoutBackgroundColour: '#121212',
```

Comment:
is there a descriptor we could add for this color? (like `gray-500` that we had before)

Reply:
Good question.  I couldn't find one dark enough in our set [here](https://github.com/code-dot-org/code-dot-org/blob/59c68befe09e4d8be5e6a886586b98efbed546bc/apps/src/music/views/theme/tokens-dark.scss#L18), nor in most of the online equivalents of this set, but maybe @sanchitmalhotra126 knows more about these...

Reply:
Hmm yeah everything in `tokens-dark.scss` was provided by Mark. Where are we getting this color (#121212) from?

Reply:
I hand-picked it!  I changed the flyout colour because the new blocks in the flyout were hard to see against the lighter grey, and this darker grey helped with legibility when the flyout is also covering workspace blocks.

Reply:
Gotcha! Yeah maybe we should just drop a comment there indicating why it needs to be so dark (can also check with Mark if we want to add this to our existing tokens at some point)

Reply:
Will do.

Also cc @markabarrett for this thread!